### DISCUSSION TITLE
Android Firefox Webmidi could not be enabled

### DISCUSSION THREAD

**Discussion by schoko11** [djipco/webmidi#336] [Q&A]
As a somekind of follow up to 
https://github.com/djipco/webmidi/discussions/334

On Android (Galaxy S20, Android 13, Firefox 109) opening up https://schoko11.github.io/KEKO-WEB/  (https://github.com/schoko11/KEKO-WEB) 
just gave an error
`TypeError: navigator.requestMIDIAccess is not a function`

on
```
const kemperMidi = WebMidi
    .enable({sysex: true})
    .then(onEnabled)
    .catch(err => alert("Error " + err));
```
when using chrome i get a request to allow the connection, but could not find it on firefox ???

Furthermore it seems that there is one in and output channel present(when setting it up like in the CME Manual), when adding a listener
function, test is not called on android (+ chrome 109) while this works on WIN10 (when sending sysex from the hardware)
```
WebMidi.inputs[0].addListener("sysex",test);
 ```

Thanks!

**Comment by djipco** [comment]
This is most likely not related to #334. It simply means there's no support for the Web MIDI API. What happens if you try in Chrome on Android?

**Reply by schoko11** [reply]
just edited my question, it works on chrome (that's fine i won't bother with firefox then). 

But i still don't get it why i couldn't get sysex messages....
